src/frontend/components/data-table/lib/component.js

```diff
@@ -588,6 +588,7 @@ class DataTableComponent extends Component {
   getConf() {
     const confData = this.el.data('config')
     let conf = {}
+    const self = this;
```

Comment:
I didn't intend to push this - I shall remove this change as it _should_ have been resolved in a different PR